po/fi.po

```diff
@@ -25,83 +27,86 @@ msgstr "Epäkelpo URL"
 
 #: ../src/backtrace.wsgi:29 ../src/log.wsgi:29 ../src/status.wsgi:30
 msgid "There is no such task"
-msgstr ""
+msgstr "Tällaista tehtävää ei ole"
 
 #: ../src/backtrace.wsgi:34 ../src/log.wsgi:34 ../src/status.wsgi:35
 msgid "Invalid password"
 msgstr "Väärä salasana"
 
 #: ../src/backtrace.wsgi:38
 msgid "There is no backtrace for the specified task"
-msgstr ""
+msgstr "Määritetylle tehtävälle ei ole jäljitysjälkeä"
 
 #: ../src/create.wsgi:60 ../src/create.wsgi:116
 msgid "Retrace server is fully loaded at the moment"
-msgstr ""
+msgstr "Jäljityspalvelin on tällä hetkellä täysin ladattu"
 
 #: ../src/create.wsgi:64
 msgid "You must use POST method"
-msgstr ""
+msgstr "Sinun on käytettävä POST-menetelmää"
 
 #: ../src/create.wsgi:68
 msgid "Specified archive format is not supported"
-msgstr ""
+msgstr "Määritettyä arkistomuotoa ei tueta"
 
 #: ../src/create.wsgi:72
 msgid "You need to set Content-Length header properly"
-msgstr ""
+msgstr "Sinun on määritettävä Content-Length:n otsikko oikein"
 
 #: ../src/create.wsgi:76
 msgid "Specified archive is too large"
-msgstr ""
+msgstr "Määritetty arkisto on liian suuri"
 
 #: ../src/create.wsgi:81
 msgid "X-CoreFileDirectory header has been disabled by server administrator"
 msgstr ""
+"Palvelimen järjestelmänvalvoja on poistanut X-CoreFileDirectory:n otsikon "
+"käytöstä"
 
 #: ../src/create.wsgi:91
 msgid "Unable to create working directory"
 msgstr "Hakemistoa ei voitu luoda"
 
 #: ../src/create.wsgi:97
 msgid "Unable to obtain disk free space"
-msgstr ""
+msgstr "Vapaata levytilaa ei saada"
 
 #: ../src/create.wsgi:101 ../src/create.wsgi:172
 msgid "There is not enough storage space on the server"
 msgstr "Palvelimella ei ole tarpeeksi vapaata tilaa"
 
 #: ../src/create.wsgi:109
 msgid "Unable to create new task"
-msgstr ""
+msgstr "Uuden tehtävän luominen epäonnistui"
 
 #: ../src/create.wsgi:121
 msgid "The directory specified in 'X-CoreFileDirectory' does not exist"
-msgstr ""
+msgstr "X-CoreFileDirector:ssa määritettyä hakemistoa ei ole olemassa"
```

Comment:
This looks like a typo. I've reported it on Weblate, but I'm merging it for now anyway.

Reply:
Finnish (and Estonian IIRC) use the colon to append declension endings to non-native words, however odd it may look. This should be ok.

Reply:
Aah, funny. But good to know!